src/tools/cephfs/cephfs-shell

```diff
@@ -536,7 +536,8 @@ exists.')
     @with_argparser(put_parser)
     def do_put(self, args):
         """
-        Copy a file to Ceph File System from Local Directory.
+        Copy given file(s) from current working directory to CephFS. Files may
+        not necessarily be regular files; can be directories too.
```

Comment:
I find `from current working directory` a bit redundant as that's always the case with any UNIX tool. How about `Copy a local file/directory to CephFS`

Reply:
sounds great.

Reply:
What about the other docstring -`Copy a file/directory from CephFS from to current working directory.`? Can't think of a way to not mention CWD here.